src/App.tsx

```diff
@@ -9,9 +9,28 @@ const App: React.FC = () => {
   return (
     <Router>
       <Navbar/>
```

Comment:
Different NavBars for public and admin, so this will need to be within the routes.